README.md

```diff
@@ -12,17 +19,42 @@ Routemaster comes with, and is automatically integration-tested against
 a Ruby client,
 [routemaster-client](https://github.com/deliveroo/routemaster-client).
 
-## Remote procedure call as an antipattern
+For advanced bus consumers,
+[routemaster-drain](https://github.com/deliveroo/routemaster-drain) can perform
+filtering of event stream
+preemptive caching of resources.
+
+## The basics
+
+Routemaster lets publisher push events into topics, and subscribers receive
+events about topics they've subscribed to.
+
+Pushing, receiving, and subscribing all happen over HTTP.
+
+![](https://www.dropbox.com/s/qbn1srjjcb8t0vh/Screenshot%202016-08-25%2016.41.54.png?raw=1)
```

Comment:
:100:

Although I'm not sure if this makes clear that a subscriber will receive batched messages. In this particular example, subscriber 2 may receive N numbers of messages belonging to 3 different topics batched in one request.